src/simplewallet/simplewallet.cpp

```diff
@@ -6756,17 +6768,17 @@ bool simple_wallet::ons_update_mapping(std::vector<std::string> args)
     }
 
     if(owner.size()) {
-      std::cout << boost::format(tr("Old Owner:        %s")) % response[0].owner << std::endl;
+      std::cout << boost::format(tr("Old Owner:        %s")) % response[0]["owner"] << std::endl;
       std::cout << boost::format(tr("New Owner:        %s")) % owner << std::endl;
     } else {
-      std::cout << boost::format(tr("Owner:            %s (unchanged)")) % response[0].owner << std::endl;
+      std::cout << boost::format(tr("Owner:            %s (unchanged)")) % response[0]["owner"] << std::endl;
     }
 
     if(backup_owner.size()) {
-      std::cout << boost::format(tr("Old Backup Owner: %s")) % response[0].backup_owner.value_or(NULL_STR) << std::endl;
+      std::cout << boost::format(tr("Old Backup Owner: %s")) % response[0]["backup_owner"] << std::endl;
       std::cout << boost::format(tr("New Backup Owner: %s")) % backup_owner << std::endl;
     } else {
-      std::cout << boost::format(tr("Backup Owner:     %s (unchanged)")) % response[0].backup_owner.value_or(NULL_STR) << std::endl;
+      std::cout << boost::format(tr("Backup Owner:     %s (unchanged)")) % response[0]["backup_owner"] << std::endl;
```

Comment:
This is optional and nlohmann with throw if it doesnt exist

Reply:
nlohmann does have a `.value("key", default)` that you can use to get the key if it exists, otherwise returning a default.